testing/trino-tests/src/test/java/io/trino/tests/TestServer.java

```diff
@@ -257,25 +258,21 @@ public void testVersionOnError()
     @Test
     public void testVersionOnCompilerFailedError()
     {
-        int numberColumns = 170;
         String tableName = "memory.default.test_version_on_compiler_failed";
         try (TestingTrinoClient testingClient = new TestingTrinoClient(server, testSessionBuilder().build())) {
             testingClient.execute("DROP TABLE IF EXISTS " + tableName);
-            testingClient.execute("CREATE TABLE " + tableName + " AS SELECT " +
-                    IntStream.range(0, numberColumns)
-                            .mapToObj(columnNumber -> format("'' AS f%s", columnNumber))
-                            .collect(joining(", ")));
-
-            String pivotQuery = "SELECT x, y " +
-                    "FROM " + tableName + " " +
-                    "CROSS JOIN UNNEST(" +
-                    IntStream.range(0, numberColumns)
-                            .mapToObj(Integer::toString)
-                            .collect(joining(", ", "ARRAY[", "]")) + "," +
-                    IntStream.range(0, numberColumns)
-                            .mapToObj(columnNumber -> format("f%s", columnNumber))
-                            .collect(joining(", ", "ARRAY[", "]")) + ") t(x, y)";
-            checkVersionOnError(pivotQuery, "TrinoException: Compiler failed(?s:.*)at io.trino.sql.gen.PageFunctionCompiler.compileProjection");
+            testingClient.execute("CREATE TABLE " + tableName + " AS SELECT '' as foo");
+
+            // This query is designed to cause a compile failure, and hopefully not run to long.
+            // The exact query is not important, just that it causes a failure.
```

Comment:
Seems redundant, this is what test method name is saying

Reply:
I like comments like this as it helps me when updating to know if the exact query shape was important.